.github/workflows/check-outdated-content.yml

```diff
@@ -0,0 +1,159 @@
+# Contribution by Cloud Native Guatemala folks :)
+# This workflow will check if a localized content is outdated or not
+# by comparing English content in the old branch and the latest branch.
+---
+name: Check outdated content
+
+on:
+  pull_request:
+
+
+jobs:
+  check-outdated-content:
+    name: Check outdated content
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout 
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0 # fetch all history for all tags and branches
+    - name: Detecting pending translations
+      shell: bash
+      run: |
+        ##### DEBUG section, this will be removed later ###########
```

Comment:
The comment says "remove later". It would be good to keep this logging functionality and enable these logs over a `--debug` flag.

Example code by an LLM (needs to be tested)
```bash
# Default debug level
DEBUG_LEVEL="info"

# Parse command-line options
while [[ "$1" != "" ]]; do
    case $1 in
        --debug)
            DEBUG_LEVEL="debug"
            ;;
        *)
            # Unknown option
            echo "Unknown option: $1"
            exit 1
            ;;
    esac
    shift
done

# Log function
_log() {
    local message="$1"
    local level="$2"

    # Determine if the message should be logged based on DEBUG_LEVEL
    case "$DEBUG_LEVEL" in
        debug)
            # Log all messages
            echo "$level: $message"
            ;;
        info)
            # Log only info and error messages
            if [[ "$level" == "info" || "$level" == "error" ]]; then
                echo "$level: $message"
            fi
            ;;
        error)
            # Log only error messages
            if [[ "$level" == "error" ]]; then
                echo "$level: $message"
            fi
            ;;
    esac
}

# Example usage:
_log "This is an info message" "info"
_log "This is a debug message" "debug"
_log "This is an error message" "error"
```

Reply:
Also, it would be spectacular if there is an `--dry-run` option. So we can see which issues would be opened without opening any. (but we dont need to add this - would be great tho 😄, very helpful debugging, i suppose). Example execution: `translation.sh <ORG> <REPO> <PR NUMBER> --dry-run --debug`

Reply:
Yes, We were thinking about to have a script instead of put this in the pipeline, is a good idea.

Reply:
@sergioarmgpl @Dianmz if you believe that it makes sense to add support for --debug and --dry-run options to the script, can you please create a task to track this as an improvement?

Reply:
I added today the debug part to this issue https://github.com/cncf/tag-env-sustainability/issues/429 @guidemetothemoon